src/scratchpads/chat_utils_rag.rs

```diff
@@ -1,721 +0,0 @@
-use std::sync::Arc;
```

Comment:
looks like this file was removed but where are those utils?

Reply:
pp_utils.rs, pp_context_files.rs, pp_plain_text.rs